transport/http/inbound.go

```diff
@@ -182,8 +182,12 @@ func (i *Inbound) start() error {
 		bothResponseError: i.bothResponseError,
 		logger:            i.logger,
 	}
-	if i.interceptor != nil {
-		httpHandler = i.interceptor(httpHandler)
+
+	// We want the first interceptor to be added last
```

Comment:
nit: I got confused with this comment, could you make it something like (not exactly) 
"Begin wrapping the `HTTP.handler` from the last interceptor as we want the first interceptor to be the root/first `HTTP.handler` wrapping the following interceptors. The last interceptor wraps the yarpc HTTP.handler. "

Reply:
+1 for this, please add more details about why these lines are important